readme.md

```diff
@@ -89,6 +89,7 @@ Feel free to play around with the code and fork this repl at [https://repl.it/@v
 - [garson](https://github.com/goliney/garson) - Build interactive config-based command-line interfaces.
 - [git-contrib-calendar](https://github.com/giannisp/git-contrib-calendar) - Display a contributions calendar for any git repository.
 - [gitgud](https://github.com/GitGud-org/GitGud) - An interactive command-line GUI for Git.
+- [Autarky](https://github.com/pranshuchittora/autarky) - Liberating disk space from file_folder node_modules.
```

Comment:
```suggestion
- [Autarky](https://github.com/pranshuchittora/autarky) - Liberating disk space from node_modules.
```

And I don't think the description explains clearly enough what it actually does. No marketing talk or tag lines.

Reply:
Thanks for the review @sindresorhus :+1: 
I have made the respective changes looking forward to your review.